homeassistant/components/sensor/wunderground.py

```diff
@@ -114,10 +123,15 @@ def __init__(self, rest, condition):
         """Initialize the sensor."""
         self.rest = rest
         self._condition = condition
+        self.entity_id = ENTITY_ID_FORMAT.format("pws_" + self._condition)
```

Comment:
Also, why would we change this and not depend on the usual resolution based on the name ?